docs-ui/stories/assets/icons/popper.tsx

```diff
@@ -29,31 +55,27 @@ const IconPopper = ({icon, setSelectedIcon, boxRef}: Props) => {
   const [isCircled, setIsCircled] = useState(icon.defaultProps?.isCircled ?? false);
   const [isSolid, setIsSolid] = useState(icon.defaultProps?.isSolid ?? false);
 
-  /**
-   * Generate and update code sample based on prop states
-   */
-  const getCodeSample = () => {
-    return `<Icon${icon.name} color="gray500" size="${size}"${
-      isCircled ? ' isCircled' : ' '
-    }${isSolid ? ' isSolid' : ' '} />`;
-  };
-  const [codeSample, setCodeSample] = useState(getCodeSample());
-  useEffect(() => {
-    setCodeSample(getCodeSample());
-  }, [size, isCircled, isSolid]);
+  const codeSample = useCallback(
+    () => getCodeSample({name: icon.name, size, isCircled, isSolid}),
+    [icon.name, size, isCircled, isSolid]
+  );
 
   /**
    * Deselect icon box on outside click
    */
-  const clickAwayHandler = e => {
-    if (e.target !== boxRef && !boxRef?.contains?.(e.target)) {
```

Comment:
This had a TS error -> property contains does not exist on RefObject as we werent actually accessing the .current